spec/models/task_spec.rb

```diff
@@ -1,5 +1,28 @@
 require 'rails_helper'
 
 RSpec.describe Task, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+	it "タイトルとステータスがあれば有効なこと" do
+		user = create(:user)
+		task = build(:task, user_id: user.id)
+		expect(task).to be_valid
+	end
+	it "タイトルがなければ無効なこと" do
+		user = create(:user)
+		task = build(:task, user_id: user.id, title: "")
+		task.valid?
+		expect(task.errors[:title]).to include("can't be blank")
+	end
+	it "ステータスがなければ無効なこと" do
+		user = create(:user)
+		task = build(:task, user_id: user.id, status: "")
```

Comment:
idではなくオブジェクトを指定する形でも作成できるかと思います。
```suggestion
		task = build(:task, user: user, status: "")
```

Reply:
ef3bd33 でidの指定からオブジェクトの指定に修正しました